data/zilliqa/assets/zil1hzt4wchy6yu9pufk07gzptsj8m3spztuqmeucs/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "EnviZion",
+    "symbol": "EVZ",
+    "web": "https://envizion.world",
+    "supply": "10000000000",
+    "decimals": 8,
+    "whitepaper": "https://docs.envizion.world",
+    "publicTeam": true,
```

Comment:
remove

Reply:
Update logo and meta.json

Reply:
that's not removed.

Reply:
removed

Reply:
I didn't say to remove the links, I commented on the publicTeam line.

Reply:
Sorry, my bad.